lib/3scale/backend/storage.rb

```diff
@@ -205,6 +208,8 @@ def cfg_sentinels_handler(options)
               end
             end.compact
 
+            # Handle role option when sentinels are validated
+            options[:role] = role if role && !role.empty?
```

Comment:
I'd ensure here we do the `to_sym`. It also avoids adding the `to_sym` to the template.

Reply:
A question that I now think about is: what if `sentinels` is parsed and returns an empty array (ie. the input is a string made of spaces or commas), would adding the `role` parameter here break the client setup?

Reply:
Regarding removing ```to_sym``` from configuration file, I aggree. But to be consistent, all other conversions should be removed as well: *to_i* and other *to_sym*. I guess this task belongs to other PR, it is out of scope.

Reply:
well, it makes sense to not introduce _yet another_ setting doing the wrong thing - if you don't fix it here, can you create a PR to clean this up?